src/olympia/ratings/models.py

```diff
@@ -279,12 +279,10 @@ def post_save(sender, instance, created, **kwargs):
             else:
                 log.info(f'{action} rating: {instance.pk}')
 
-            # For new ratings - not replies - and all edits (including replies
-            # this time) by users we want to insert a new ActivityLog.
-            new_rating_or_edit = not instance.reply_to or not created
-            if new_rating_or_edit:
-                action = amo.LOG.ADD_RATING if created else amo.LOG.EDIT_RATING
-                activity.log_create(action, instance.addon, instance)
+            # For new ratings, replies, and all edits by users we want
+            # to insert a new ActivityLog.
+            action = amo.LOG.ADD_RATING if created else amo.LOG.EDIT_RATING
+            activity.log_create(action, instance.addon, instance)
```

Comment:
Lets log a different action for replies like mat [suggested](https://github.com/mozilla/addons-server/issues/20168#issue-1528827756) - you can just add a new class to activity.py for `REPLY_RATING` (or whatever)

Reply:
Is there any structure for the log IDs in activity.py or any free one will do?

Reply:
There probably should be, but there isn't any particular structure. Just add it at the end and pick the next id number based on the last id + 1.